src/server/pwa/offline.jsx

```diff
@@ -1,67 +0,0 @@
-/*
```

Comment:
Why was this middleware deleted?

Reply:
because it was moved into the react html plugin (see https://github.com/americanexpress/one-app/pull/856)